components/button/button.tsx

```diff
@@ -50,9 +49,16 @@ export interface BaseButtonProps {
   ghost?: boolean;
 }
 
-export type AnchorButtonProps = BaseButtonProps & React.AnchorHTMLAttributes<HTMLAnchorElement>;
+export type AnchorButtonProps = {
+  href: string;
+  target?: string;
+  onClick?: React.MouseEventHandler<HTMLAnchorElement>;
```

Comment:
Are these props already included in `React.AnchorHTMLAttributes<HTMLAnchorElement>`?

Reply:
Yeah, you are right. But in `<Button />` I change the 'href' prop as required to distinguish which kind of tag should be rendered.

'target' and 'onClick' are included in `React.AnchorHTMLAttributes<HTMLAnchorElement>`,  we cloud remove them, but these props are list in the API [documention](https://ant.design/components/button-cn/#API), I think it is ok to pick them out of React.AnchorHTMLAttributes<HTMLAnchorElement> personally and it is common in other Compoents.